backend/src/neuroagent/mcp.json

```diff
@@ -0,0 +1,12 @@
+{
+    "tavily": {
+      "command": "npx",
```

Comment:
So if we want to have this server in production then we need to update our Dockerfile and make sure `node` is installed.